packages/analytics/analytics-chart/src/components/chart-types/StackedBarChart.vue

```diff
@@ -572,8 +572,8 @@ const handleChartClick = () => {
 }
 
 .axis-tooltip {
-  background-color: var(--grey-600, #3c4557);
-  color: var(--white);
+  background-color: $kui-color-text-neutral-stronger;
+  color: $kui-color-text-inverse;
   padding: 5px 10px 5px 10px;
```

Comment:
Won't `kui-space-20` and `kui-space-40` work here?

Reply:
This was pushed up